ui/selectors/multichain-accounts/account-tree.ts

```diff
@@ -41,10 +41,11 @@ import { getSanitizedChainId, extractWalletIdFromGroupId } from './utils';
  * @param state - Redux state.
  * @param state.metamask - MetaMask state object.
  * @param state.metamask.accountTree - Account tree state object.
- * @returns Account tree state.
+ * @returns Account tree state, or default empty state if not available.
  */
 export const getAccountTree = createDeepEqualSelector(
-  (state: MultichainAccountsState) => state.metamask.accountTree,
+  (state: MultichainAccountsState) =>
+    state.metamask.accountTree ?? { wallets: {}, selectedAccountGroup: null },
```

Comment:
Not sure how we should handle empty states? @hmalik88 I was getting an error that `selectedAccountGroup` was undefined.